Automattic/wp-calypso
# VideoPress Intent: record waitlist in Crowdsignal

Related to # https://github.com/Automattic/greenhouse/issues/1834

## Proposed Changes

* add a POST request to crowdsignal to record email addresses in a survey
* add a success message after email is submitted
* add validation to the email field

| email field | success message |
| - | - |
| <img width="766" alt="SCR-20230830-iuap" src="https://github.com/Automattic/wp-calypso/assets/4081020/b9bca781-6d4c-4632-ba95-a34cb3fcfcad"> | <img width="713" alt="SCR-20230830-iuny" src="https://github.com/Automattic/wp-calypso/assets/4081020/1a2eed9b-6df6-4526-8c40-e6165c54b064"> |


## Testing Instructions



* `yarn && yarn start`
* `/setup/videopress`
* Open the Channels or "upload a Video" items
* enter an invalid email, try to submit. You shall be stopped
* enter a valid email, try to submit. You should see a "thank you" message
* Check the survey results to see that your valid email was recorded
* close the modal or refresh the page, reopen the same item, the waitlist should still show the "thank you" message
* open the other Item, the waitlist shouldn't show the "thank you" message because it has not been submitted yet (unless you already did previously)

## Pre-merge Checklist
